### ISSUE TITLE
How does label0 or label1 matter for testing the style accuracy?

### ISSUE THREAD

**Issue by chejmadi** [shrimai/Style-Transfer-Through-Back-Translation#21]
So we translated the French republican test set using the democratic decoder/generator and ran this output through cnn_translate. We gave the target as "democratic" and the labels as label0 = republican and label1 = democratic. My question is, should label1 specifically be the target? If I did the analogous experiment for republican output, would I need to give the label1 as republican (with target being republican) and label0 as democratic?

**Comment by shrimai** [comment]
No. label0 and label1 are the labels with which the classifiers are trained. They are fixed for the pre-trained classifiers. For the political slant task, label0 = republican and label1 = democratic. If you wanted to test against target as republican, you say -target republican and label0 = republican and label1 = democratic.
